src/development/LocalTest/LocalTest.csproj

```diff
@@ -16,4 +16,8 @@
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.4" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Do we need to include it here?

Reply:
No, it's unnecessary, removing. These were added because I created the folders before the class.